components/renku_data_services/project/orm.py

```diff
@@ -48,12 +48,15 @@ class ProjectORM(BaseORM):
     # NOTE: The project slugs table has a foreign key from the projects table, but there is a stored procedure
     # triggered by the deletion of slugs to remove the project used by the slug. See migration 89aa4573cfa9.
     slug: Mapped["EntitySlugORM"] = relationship(
-        # NOTE: joined lazy relationship can cause duplicates to be returned
-        lazy="selectin",
+        lazy="joined",
         init=False,
         repr=False,
         viewonly=True,
         back_populates="project",
+        # NOTE: If the data_connector ID is not null below then multiple joins are possible here
+        # since an entity slug for data connector owned by a project will and an entity slug for a project
```

Comment:
you accidentally a word
```suggestion
        # since an entity slug for data connector owned by a project and an entity slug for a project
```

At least I think this is right this way